### PR TITLE
Updates and Bug Fixes

### PR DESCRIPTION
**Pull request by guyrish**
Added deities.
Corrected spelling errors in the equipmods and feats files.
Corrected a class access to a spell. Added feat check to the Mother Cyst
/ Necrotic X spells.

### CODE REVIEW

File: data/35e/wizards_of_the_coast/supplement/libris_mortis/lm_deities.lst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+
+Afflux			DOMAINS:Deathbound,Evil,Knowledge,Undeath		DESC:Afflux (M)			DEITYWEAP:Sword (Short)		ALIGN:NE	PANTHEON:Undead		TYPE:Lesser
+Doresain		DOMAINS:Chaos,Evil,Hunger				DESC:Doresain (M)		DEITYWEAP:Scimitar		ALIGN:CE	PANTHEON:Undead		TYPE:Demigod
+Evening Glory		DOMAINS:Chaos,Charm,Good				DESC:Evening Glory (F)		DEITYWEAP:Dagger		ALIGN:TN	PANTHEON:Undead		TYPE:Lesser
+Nerull			DOMAINS:Death,Evil,Trickery				DESC:Nerull (M)			DEITYWEAP:Scythe		ALIGN:NE	PANTHEON:Undead		TYPE:Greater
+#Orcus			DOMAINS:Chaos,Darkness,Death,Evil			DESC:Orcus (M)			DEITYWEAP:Mace (Light)|Mace (Heavy)		ALIGN:CE	PANTHEON:Undead		TYPE:Demon Lord
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/supplement/libris_mortis/lm_deities.lst]: Why is this entry commented out?

File: data/35e/wizards_of_the_coast/supplement/libris_mortis/lm_spells.lst [hunk 2]
```diff
@@ -60,3 +60,14 @@ Undead Bane Weapon									TYPE:Divine			CLASSES:Paladin=3|Cleric=4									SCHO
...
+Necrotic Awareness.MOD		PREFEAT:1,Mother Cyst
+Necrotic Bloat.MOD		PREFEAT:1,Mother Cyst
+Necrotic Burst.MOD		PREFEAT:1,Mother Cyst
+Necrotic Cyst.MOD		PREFEAT:1,Mother Cyst
+Necrotic Domination.MOD		PREFEAT:1,Mother Cyst
+Necrotic Empowerment.MOD	PREFEAT:1,Mother Cyst
+Necrotic Eruption.MOD		PREFEAT:1,Mother Cyst
+Necrotic Scrying.MOD		PREFEAT:1,Mother Cyst
+Necrotic Tumor.MOD		PREFEAT:1,Mother Cyst
+Necrotic Termination.MOD	PREFEAT:1,Mother Cyst
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/supplement/libris_mortis/lm_spells.lst]: PREFEAT is deprecated, these entries should be PREABILITY:1,CATEGORY=Feat,

**Comment by BahamutDragon** [comment]
Option to override with newer sources should be used. Include it please.

**Comment by FerretDave** [comment]
Is it worth setting that option directly in the spell compendium?                                                                                                                                                                                                                                                                                                                                        CheersDave     oxxxxxxx{;;;;;;;;;;;;;;;;;;;;;;;;;;;;;>                                                                                                                                                                                                                From: BahamutSent: Monday, 8 February 2016 07:13To: BahamutDragon/pcgenReply To: BahamutDragon/pcgenSubject: Re: [pcgen] Updates and Bug Fixes (#382)Option to override with newer sources should be used. Include it please.

—Reply to this email directly or view it on GitHub.

**Comment by BahamutDragon** [comment]
The option should be set in the pcgen program. Which is why a conflict message occurs.

I can see how the intent of my reply was unclear. To clarify, my intention was to have him remove the comment from in front of Orcus.

Best,

**Comment by FerretDave** [comment]
I like the idea of Orcus being commented out! If only deicide was so easy
in-game- a single 'hash' character does more damage than a +5 vorpal blade
;-)

Spell Compendium - as it does override so may other sources, and does need
that 'newer sources overide older' option, my suggestion is that the PCC
for the spell compendium itself actually turn that option on. (Though I
appreciate that that in itself requires an in-app setting to actually allow
sources to set options...)

It's just a step closer to ensuring that the required option is set.

Cheers
Dave

**Pull request merged**